app/server.py

```diff
@@ -256,16 +256,54 @@ def delete_product(id):
 
 @app.route('/inventory/products', methods=['POST'])
 def create_products():
-  """ create a product with restock level.
-  This method will create a storage for a new product
-
-  Args: data with restock_level provided.
-
-  Returns:
-
-    response: create successful message with status 201 if succeeded,
-              the auto assigned product id and location id should also be presented
-              or invalid create with status 400 if the create request violates any limitation
+  """
+    Creates a Product
+    This endpoint will create a Product based the data in the body that is posted
+    ---
+    tags:
+      - Products
+    consumes:
+      - application/json
+    produces:
+      - application/json
+    parameters:
+      - in: body
+        name: body
+        required: true
+        schema:
+          id: data
+          required:
+            - restock_level
+          properties:
+            restock_level:
+              type: integer
+              description: total space allocated for the product
```

Comment:
@plzhang I feel like the description is confusing..
Maybe it's better to call it something like "the quantity limit for this product" ?